src/relay/GuaranteeNeededResolver.js

```diff
@@ -107,6 +108,12 @@ class GuaranteeNeededResolver extends React.Component<Props> {
     );
 
     if (showGuaranteeChat !== this.props.showGuaranteeChat) {
+      if (showGuaranteeChat) {
+        simpleTracker('smartFAQBookingOverview', {
+          action: 'chatEnabled',
```

Comment:
can you add this tracker to the README.md https://github.com/kiwicom/smart-faq#analytics so we have an updated list of all trackers running.

Reply:
added